packages/core/src/__tests__/ceramic-feed.test.ts

```diff
@@ -131,15 +124,11 @@ describe('Ceramic feed', () => {
     // process anchor
     await TestUtils.anchorUpdate(ceramic1, stream)
 
-    expect(feed.length).toEqual(3) // create + anchor request + anchor update
+    expect(feed.length).toEqual(2) // genesis + anchor commits
     // between and request anchor
```

Comment:
what does this comment mean? @JulissaDantes

Reply:
The 2 elements inside the feed are `2 commits = 1 genesis commit + 1 anchor commit`

Reply:
changed to `2 commits = 1 genesis commit + 1 anchor commit`

Reply:
nice, improved comment on the previous line.  Can we just delete the comment on this line?